src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -85,6 +85,7 @@ public static class RpcWorkerConstants
         public const string WorkerIndexingEnabled = "WORKER_INDEXING_ENABLED";
         public const string WorkerIndexingDisabledApps = "WORKER_INDEXING_DISABLED_APPS";
         public const string RevertWorkerShutdownBehavior = "REVERT_WORKER_SHUTDOWN_BEHAVIOR";
+        public const string RevertWorkerShutdownForUnhealthyHost = "REVERT_WORKER_SHUTDOWN_FOR_UNHEALTHY_HOST";
```

Comment:
When naming feature flags, I find it easier to understand the name if it's "positive". E.g. in this case ShutdownStandbyWorkerChannelsOnHostShutdown or something and you can use the GetFeatureOrDefault overload that allows you to specify your true default. For an example, see SwtAuthenticationEnabled.

Reply:
done.